fastchat/train/train_lora.py

```diff
@@ -102,11 +100,16 @@ def train():
         lora_args,
     ) = parser.parse_args_into_dataclasses()
 
-    device_map = None
+    if model_args.flash_attn == True:
+        print("Using Flash Attention!")
+        replace_llama_attn_with_flash_attn()
+
+    device_map = "auto"
```

Comment:
I believe this should be `None` because if we use ZeRO3, we can't pass in a `device_map`. Would you be able to double-check that this works with ZeRO3?